### ISSUE TITLE
Rephrase text/info

### ISSUE THREAD

**Issue by svx** [plone/ploneorg.core#142]
The phrasing on https://plone.org/download/releases/5.0.4 in the red box about virtualbox/vagrant may be confusing.

![download_plone](https://cloud.githubusercontent.com/assets/358860/15185790/cf0f7364-179a-11e6-841a-6fca68c93ac3.png)

Clarification: We have a vagrant kit, which is build with a so called 'vagrant provider' that is all what we have, we do not have a virtualbox image, at least no one which we currently provide.

Reading the text in the red box, someone could assume that it is possible to just download and do a import into virtualbox.

Further, it would be nice if the boxes could have the same size, much smoother on the eyes :)

Thank you for all your awesome work !

**Comment by vikas-parashar** [comment]
IMO, we can get rid of the list styling(dots) also. 
![rsz_screenshot_from_2016-05-12_000828](https://cloud.githubusercontent.com/assets/5694967/15192348/667c4674-17d6-11e6-8f3e-55780639b2a5.jpg)

**Comment by polyester** [comment]
@vikas-parashar css-wizardry patches gladly accepted; if you manipulated it through firebug or other devtools just post them here and I'll gladly find the right file to put them in. Your version looks much better, so let's get this in !

**Comment by vikas-parashar** [comment]
Yeah, I did this using dev tools. Here's the code, use it if it's works properly on all devices.

Removal of list styling:
`ul{
    list-style: none;
}`

Resizing of second box( This depends on text so need to changed if the text changes):
`a{
    padding: 10px 50px;
}`

**Comment by svx** [comment]
fixed ! thanks !

**Comment by gforcada** [comment]
@svx should be closed then?
